test/unit/kubernetes-deploy/kubeclient_builder_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'kubernetes-deploy/kubeclient_builder'
+
+class KubeClientBuilderTest < KubernetesDeploy::TestCase
+  include KubernetesDeploy::KubeclientBuilder
+
+  def setup
+    Kubeclient::Client.any_instance.stubs(:discover)
+    super
+  end
+
+  def test_build_client_from_multiple_config_files
+    old_config = ENV['KUBECONFIG']
+    begin
```

Comment:
Same comment about the begin/end here (sorry, should have commented here at the same time

Reply:
Updated.